src/bindings/swig/python/examples/example_keyset.py

```diff
@@ -23,6 +23,17 @@
 ks1.remove("user:/key6")
 print("")
 
+print("Alternatively you can use pop")
+ks1.extend(list_of_keys)
+print(ks1.pop())
+print(ks1.pop())
```

Comment:
You should mention (in a comment) what `pop` actually does.

Reply:
added a comment in [2fc82a9](https://github.com/ElektraInitiative/libelektra/pull/4321/commits/2fc82a98436114b8085556cf85fcc968073291bc)